src/framework/preferences/html/PrefsEditorTemplate-speak.html

```diff
@@ -1,11 +1,10 @@
 <h2><span class="fl-icon-speak"></span><span class="flc-prefsEditor-speak-label heading-text"></span></h2>
-
 <div class="fl-prefsEditor-onoff">
     <input type="checkbox" id="speak-choice" class="flc-prefsEditor-speak" />
     <label for="speak-choice">
+    	<p class="flc-prefsEditor-speak-choice-label"></p>
```

Comment:
`<p>` tags are not valid descendants of a `<label>`. Basically you can't use block level elements inside of an inline element. You can use this validator to check the markup, but you don't need to worry about things that will be adjusted by the javascript (e.g. the empty heading).

https://validator.w3.org/nu/